### PR TITLE
Creation of the front page html and css

### PR DESCRIPTION
**Pull request by ghost**
Please do consider that the front page is a starting point and need to be worked more.

### CODE REVIEW

File: Pictures/legoMan.svg [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+	<path d="M-765.7,1627.9c0,0.6,0,1.1,0,1.6h-0.9l-0.1-0.9h0c-0.3,0.5-0.9,1.1-1.9,1.1c-0.9,0-2-0.5-2-2.5v-3.4h1.1v3.2
+		c0,1.1,0.3,1.8,1.3,1.8c0.7,0,1.2-0.5,1.4-1c0.1-0.2,0.1-0.3,0.1-0.5v-3.6h1.1L-765.7,1627.9L-765.7,1627.9z"/>
+	<path d="M-762.9,1626.4L-762.9,1626.4c0.2-0.2,0.4-0.5,0.5-0.7l1.7-2h1.3l-2.2,2.4l2.6,3.4h-1.3l-2-2.8l-0.5,0.6v2.2h-1v-8.5h1
+		L-762.9,1626.4L-762.9,1626.4z"/>
+</g>
+<g>
+	<polygon class="st16" points="-820,1121.1 -820,1132.3 -866.1,1139.3 	"/>
+	<polygon class="st16" points="-817.2,1121.1 -817.2,1132.3 -771.2,1139.3 	"/>
+</g>
+</svg>
```
Comment (spykins) [inline Pictures/legoMan.svg]: Is this file (legoMan.svg) being used anywhere?

**Review by spykins — CHANGES_REQUESTED** [review]

File: Pictures/legoMan.svg (hunk 1 above)
Comment (ghost) [reply Pictures/legoMan.svg]: The file as well as the folder must been deleted. It is my mistake. There is already an other folder in assets that take care of our needs.

**Comment by ghost** [comment]
Hey Wale, what is missing in order to accept this pull request?

File: assets/css/styles.css [hunk 2]
```diff
@@ -0,0 +1,457 @@
...
+
+}
+
+
+
+/*--------------------
+ 	Our solution Section
+---------------------*/
+
+.оur-solution{
```
Comment (spykins) [inline assets/css/styles.css:232]: Please where is this being used?

**Review by spykins — CHANGES_REQUESTED** [review]

File: assets/css/styles.css (hunk 2 above)
Comment (ghost) [reply assets/css/styles.css:232]: It is in the html file section our-solution

**Review by spykins — APPROVED** [review]

**Comment by spykins** [comment]
@Tommycioc one last thing, you have sent this pull request to be merged to master, can you please switch the pr to point to `develop` instead

**Review by spykins — APPROVED** [review]

**Pull request merged**